### PR TITLE
Lint version

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: tests/test_lint_recipe.py [hunk 1]
```diff
@@ -431,6 +431,33 @@ def test_outputs(self):
         meta = OrderedDict([['outputs', [{'name': 'asd'}]]])
         lints = linter.lintify(meta)
 
+    def test_noarch_python_pip(self):
```
Comment (isuruf) [inline tests/test_lint_recipe.py]: It seems I forgot to add this in the previous PR. cc @ocefpaf

**Comment by jakirkham** [comment]
LGTM

File: conda_smithy/lint_recipe.py [hunk 2]
```diff
@@ -256,6 +257,14 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
                     lints.append("`noarch: python` packages should use pip. "
                                  "See https://conda-forge.org/docs/meta.html#use-pip")
 
+    # 19: check version
+    if package_section.get('version') is not None:
```
Comment (CJ-Wright) [inline conda_smithy/lint_recipe.py]: Is there ever a situation where we don't have a version in the `package_section`?

Comment (isuruf) [reply conda_smithy/lint_recipe.py]: There can be a situation where it is not in a bad recipe.

**Pull request merged**